### PR TITLE
Screen class

### PR DESCRIPTION
**Pull request by VeckoTheGecko**
Discussing the screen class implementation

### CODE REVIEW

**Comment by JRavi2** [comment]
Still in progress.
Currently the horizontal screen transition is working. Vertical transition still needs debugging.
Will be ready for review/merge once the vertical transition is complete

File: src/screen.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            self.screens[9].insert(screen_x, self.BOX_PADDING_Y-1, ["─"])
+        for screen_y in range(0, self.BOX_PADDING_Y-1):
+            self.screens[9].insert(self.BOX_PADDING_X-1, screen_y, ["│"])
+
+    def insertEntity(self, entity: Entity) -> None:
+        """Inserts a object of entity class to its specified point in the current screen.
+
+        :param person: Object, imported from entities.py
+        """
+        self.screens[self.current_screen].insert_entity(entity)
```
Comment (VeckoTheGecko) [inline src/screen.py:80]: Would it be beneficial to extend this to allow for an insertion on an arbitrary screen, but defaulting to the current one? Would be good in order to initialise the scenes on the different screens. What do you think?
```suggestion
    def insertEntity(self, entity: Entity, screen_number:int = self.current_screen) -> None:
        """Inserts a object of entity class to its specified point on a specified screen (defaults to current screen)

        :param person: Object, imported from entities.py
        """
        self.screens[screen_number].insert_entity(entity)
```

Comment (JRavi2) [reply src/screen.py:80]: Yup, that would be better.
Thought I think further changes to screen class should be continued in #18

**Comment by JRavi2** [comment]
Changes merged in #18

**Pull request closed without merging**